docs/topics/whatsnew-eap.md

```diff
@@ -102,11 +91,49 @@ We would appreciate any feedback you may have!
   on [our issue tracker](https://kotl.in/issue).
 * [Enable the **Send usage statistics** option](https://www.jetbrains.com/help/idea/settings-usage-statistics.html) to
   allow JetBrains to collect anonymous data about K2 usage.
+* 
+## New directory for Kotlin data in Gradle projects
+
+In Kotlin 1.8.20, the Kotlin Gradle plugin started to store its data in the Gradle project cache directory: `<project-root-directory>/.gradle/kotlin`.
+However, the `.gradle` directory is reserved for Gradle only, and as a result it's not future-proof. To solve this, in 
+Kotlin 2.0.0-Beta2 we store Kotlin data in your `<project-root-directory>/.kotlin` by default. We will continue to store
+some data in `.gradle/kotlin` directory for backward compatibility.
+
+> With this change, you may need to add the `.kotlin` directory to your project's `.gitignore` file.
+>
+{type="warning"}
+
+There are new Gradle properties so that you can configure a directory of your choice and more:
+
+| Gradle property                                     | Description                                                                                                  |
+|-----------------------------------------------------|--------------------------------------------------------------------------------------------------------------|
+| `kotlin.project.persistent.dir`                     | Configures the location where your project-level data is stored. Default: `<project-root-directory>/.kotlin` |
+| `kotlin.project.persistent.dir.gradle.disableWrite` | A boolean value that controls whether writing data to the `.gradle` directory is disabled. Default: false    |
+
+Add these properties to the `gradle.properties` file in your projects for them to take effect.
+
+## What to expect from upcoming Kotlin EAP releases
+
+The upcoming 2.0.0-Beta3 release will introduce increased stability to the K2 compiler.
```

Comment:
Sorry, I don't understand. "will introduce increased stability to" is already present. Or did you mean: "will increase stability of" in favor of "will introduce increased stability to". So you prefer "of" instead of "to"?

Reply:
I meant change the first one for the second one, sorry. Mostly I am in favor of removing the word “introduce”.

Reply:
Ahhh I see! Applied 👍